code/modules/antagonists/_common/antag_datum.dm

```diff
@@ -477,16 +479,31 @@ GLOBAL_LIST_EMPTY(antagonists)
 /datum/action/antag_info
 	name = "Open Antag Information:"
 	button_icon_state = "round_end"
+	/// The antag datum that this button interacts with.
 	var/datum/antagonist/antag_datum
 
 /datum/action/antag_info/New(Target, datum/antagonist/antag_datum)
 	. = ..()
 	src.antag_datum = antag_datum
 	name += " [antag_datum.name]"
```

Comment:
The info button's target is the mob's current body right? That means if the mob's body qdel's, the antag button should qdel too.

Is there a reason the target shouldn't be the antag datum instead? Changing owners is easy, but changing targets is kinda a headache

Reply:
No idea if there's a reason or not, i'll do it